stylesheet/colors.css

```diff
@@ -0,0 +1,88 @@
+:root {
+
+    /* color variables */
+    
+    --clr-primary: #ffffff;
+    --clr-off-primary: #CCCAC9;
+    --clr-secondary: #000000;
+    --clr-off-secondary: #323232;
+    --clr-accent: #e91c23;
+    
+    --clr-warning: #FFCE00;
+    --clr-success: #4caf50;
+    --clr-error: #f44336;
+}
+
+
+/* colors for text  */
+
+/* primary text is dark and secondary text is light */
+
+.txt-primary {
+    color: var(--clr-secondary);
+}
+
+.txt-secondary {
+    color: var(--clr-primary)
+}
+
+.txt-off-primary {
+    color: var(--clr-off-secondary)
+}
+
+.txt-off-secondary {
```

Comment:
somewhere you are using semicolon at the end of CSS property, somewhere you are not using it
make it consistent, you should use it everywhere to make code a bit consistent

Reply:
thanks for pointing out. i have added the semicolons.